internal/imageservice/imageservice.go

```diff
@@ -0,0 +1,35 @@
+package imageservice
+
+import (
+	"fmt"
+
+	"github.com/openshift/assisted-service/models"
+)
+
+type BootArtifactURLs struct {
+	KernelURL string
+	RootFSURL string
+	InitrdURL string
+}
+
+func GetBootArtifactURLs(baseURL, imageID string, osImage *models.OsImage, insecure bool) (*BootArtifactURLs, error) {
```

Comment:
I think this can go in the same file as the other functions it's calling.

Reply:
Moved it into that file, thank you!